CMakeLists.txt

```
@@ -0,0 +1,43 @@
+# set required cmake version
+cmake_minimum_required(VERSION 3.19)
+
+# This avoids googletest complaining that this (IPO) policy is not set
+cmake_policy(SET CMP0069 NEW)
+set(CMAKE_POLICY_DEFAULT_CMP0069 NEW)
+
+project(
+  gr_dd_eval
```

```suggestion
  DDEval
```
I'd propose to leave the `gr` part out of naming schemes as much as possible. In the end, we will probably move this repository to the cda-tum organisation. I think `DDEval` seems like a nice analogy to `DDPackage`.